tests/test_examples.py

```diff
@@ -126,9 +126,9 @@ def test_partial_scatter(self):
         try:
             fail()
         except cwltool.factory.WorkflowStatus as e:
-            self.assertEquals('sha1$e5fa44f2b31c1fb553b6021e7360d07d5d91ff5e', e.out["out"][0]["checksum"])
+            self.assertEquals('sha1$356a192b7913b04c54574d18c28d46e6395428ab', e.out["out"][0]["checksum"])
```

Comment:
Hmm. Earlier we were writing \n and that gave checksum error So I removed \n and added this new checksum. But now since we have changed all `w` mode to `wb`. It would be interesting to check if it works. I think it should work without this change in checksum. I will check